modules/local/collect_genotypes.nf

```diff
@@ -0,0 +1,52 @@
+process COLLECT_GENOTYPES {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::python=3.8.3"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.8.3' :
+        'biocontainers/python:3.8.3' }"
+
+    input:
+    tuple val(meta), path(geno), path(snp), path(ind)
+
+    output:
+    tuple val(meta), path("*.geno"), path("*.snp"), path("*.ind")  , emit: collected
```

Comment:
a) These files only ever make sense together. It would be an unnecessary headache to spit them out separately and force them to align correctly before using them in any other modules.
b) This is the cardinality of the input.
c) This is the cardinality expected by downstream modules (e.g. [eigenstratsnpcoverage](https://github.com/nf-core/eager/blob/156e132f2f1c47f0893bf0b60eaad66b06b78b8d/modules/nf-core/eigenstratdatabasetools/eigenstratsnpcoverage/main.nf) )